Gio-2.0.gir

```diff
@@ -74098,6 +74098,19 @@ g_tls_certificate_new_from_pem().</doc>
           </parameter>
         </parameters>
       </constructor>
+      <constructor name="new_from_file_with_password" c:identifier="g_tls_certificate_new_from_file_with_password" throws="1">
+        <return-value transfer-ownership="full">
+          <type name="TlsCertificate" c:type="GTlsCertificate*"/>
+        </return-value>
+        <parameters>
+          <parameter name="file" transfer-ownership="none">
+            <type name="utf8" c:type="const gchar*"/>
```

Comment:
It's marked as `(type filename)` as it should, but that doesn't end up here for some reason. Probably same problem as above?